packages/fxa-payments-server/src/routes/Product/SubscriptionRedirect/index.tsx

```diff
@@ -1,33 +1,44 @@
-import React from 'react';
+import React, { useEffect } from 'react';
 import { Plan } from '../../../store/types';
 
 import './index.scss';
 
+// Mapping of product IDs to post-subscription redirect URLs
+// TODO: Move / merge in some kind of configuration variable?
```

Comment:
A useful follow-up might be to support some server-side configuration for mapping product ID to redirect URL. Not entirely unlike how we map product ID to subscription capabilities - [although *that* config happens over on auth-server](https://github.com/mozilla/fxa/blob/master/packages/fxa-auth-server/config/dev.json#L95).

Reply:
Maybe we could pass the config JSON into the page inside a template tag?

Reply:
Yeah, we already pass config into a meta tag. There's just a bit of work in the plumbing and types to add a new config setting